### PR TITLE
P&W Property Sets fixes 

### LINKED ISSUE #109: WP3-PropertyHarmonization - PortsAndWaterways properties
You will find the HarmonizationExcel here:
[HarmonizationCases_30-06-2021.xlsx](https://github.com/bSI-InfraRoom/IFC-Specification/files/6739891/HarmonizationCases_30-06-2021.xlsx)

1. Pset_MaintenanceTriggerPerformance -> MaintenanceLevel_1iqiV1Bdv9kP1zFmTnxXvA Could we change the name of the Property to MaintenanceLevelPerformance for instance, to avoid having two properties with the same name (even if one is an enumeration and the other one a singlevalue). **HarmonizationCase1=76**

2. Pset_EnergyRequirements -> EnergySource_1mlnyMavn0XvUzU0A9esFc  Could we change the name of the property to EnergySourceType for instance, to avoid having two properties with the same name (even if one is an enumeration and the other one a singlevalue).  **HarmonizationCase1.1=233**

### LINKED ISSUE #327: [P&W] Property issues
I would also use this issue to point out a couple more. Forgot a couple of times now to post these:
1. https://github.com/bSI-InfraRoom/IFC-Specification/tree/develop/IFC4x3/Properties/h/HydraulicConductivity_0n5hy7wA9CGeRruzAkgVAG
and
https://github.com/bSI-InfraRoom/IFC-Specification/tree/develop/IFC4x3/Properties/p/PwaveVelocity_25QNLFrcX23AOU3hgpu9do
also
https://github.com/bSI-InfraRoom/IFC-Specification/blob/develop/IFC4x3/Properties/s/SwaveVelocity_0Zt9yzVWjF3QUsVuryk8eT/DocProperty.xml
should be `IfcLinearVelocityMeasure`
2. https://github.com/bSI-InfraRoom/IFC-Specification/tree/develop/IFC4x3/Properties/l/LoadbearingCapacity_2r2w0IRgj2ZPPZHhlnoADU
maybe a typo in the name and the PrimaryDataType should be `IfcPressureMeasure`

_Originally posted by @SergejMuhic in https://github.com/bSI-InfraRoom/IFC-Specification/issues/326#issuecomment-991717860_

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
P&W Property set fixes highlighted during deliverable import.

Fixes #327 
Fixes P&W responsibilities of #326

### CODE REVIEW

**Comment by AlexBrad1eyCT** [comment]
@SergejMuhic a quick once over would be appreciated so it is double checked!

**Review by SergejMuhic — APPROVED** [review]

File: IFC4x3/Properties/l/LoadBearingCapacityArea_2r2w0IRgj2ZPPZHhlnoADU/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Estimate or design maximum load bearing capacity of the defined surface/structure (e.g. soil stratum or planar slab). represented as a force per unit area measurement.
```
Comment (pjanck) [inline IFC4x3/Properties/l/LoadBearingCapacityArea_2r2w0IRgj2ZPPZHhlnoADU/Documentation.md]:
```suggestion
Estimate or design maximum load bearing capacity of the defined surface/structure (e.g. soil stratum or planar slab). Represented as a force per unit area measurement.
```

**Review by pjanck — APPROVED** [review]
I wonder about the `ApplicableType` correctness, but am in no position to propose any improvement / change.

**Comment by AlexBrad1eyCT** [comment]
For Some reason PSet_MarineVehicleCommon got changed to a QTO? but not the title, although it is full of dimensional values these are not derived values from the entities geometry (which is my opinion on what a QTO is unless this has changed...) they are the standardized design values set for each size class of marine vehicle hense PSet.

I have sorted this out for you @SergejMuhic!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_MarineVehicleCommon/DocPropertySet.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarineVehicleCommon" UniqueId="ffe029d5-901e-429b-abb7-f9b61a7cf52b" ApplicableType="IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.VEHICLEMARINE.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.VEHICLEMARINE.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarineVehicleCommon" UniqueId="ffe029d5-901e-429b-abb7-f9b61a7cf52b" ApplicableType="IfcVehicle/VEHICLEMARINE,IfcVehicleType/VEHICLEMARINE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (pjanck) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_MarineVehicleCommon/DocPropertySet.xml]: I'd remove Type's applicability.

**Review by pjanck — CHANGES_REQUESTED** [review]
Thanks for the fast response!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_MarineVehicleCommon/DocPropertySet.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarineVehicleCommon" UniqueId="ffe029d5-901e-429b-abb7-f9b61a7cf52b" ApplicableType="IfcVehicle/VEHICLEMARINE" PropertySetType="QTO_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarineVehicleCommon" UniqueId="ffe029d5-901e-429b-abb7-f9b61a7cf52b" ApplicableType="IfcVehicle/VEHICLEMARINE,IfcVehicleType/VEHICLEMARINE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (pjanck) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterwaysDomain/PropertySets/Pset_MarineVehicleCommon/DocPropertySet.xml]:
```suggestion
<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MarineVehicleCommon" UniqueId="ffe029d5-901e-429b-abb7-f9b61a7cf52b" ApplicableType="IfcVehicle/VEHICLEMARINE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Perhaps easier with proposing changes :)

**Comment by SergejMuhic** [comment]
Thanks @AlexBrad1eyCT looks ready for merging (apart from the type applicability). One thing I notices, there were 29 changed files before, now there are 16. Is this just a consequence of the merge?

**Comment by SergejMuhic** [comment]
Think this one fixes #109 also.

**Comment by AlexBrad1eyCT** [comment]
@SergejMuhic, yes the reduced file changes is becuase some of the corrections had been addressed in the develop branch hence the merge conflicts so merging the most recent develop reduced the change number.

@SergejMuhic, @pjanck 
I am unsure about this issue with the Type applicability? has the prevailing opinion changed around type entities and PSET_TYPEDRIVENOVERRIDE, i have always been of the view that Psets with PSET_TYPEDRIVENOVERRIDE should be connected to the type and the instance entity. have we moved to a situation that assumes the type applicability based on the instance value seems a bit woolly to be to not use explicit notation in  the standard??

**Comment by SergejMuhic** [comment]
> I am unsure about this issue with the Type applicability? has the prevailing opinion changed around type entities and PSET_TYPEDRIVENOVERRIDE, i have always been of the view that Psets with PSET_TYPEDRIVENOVERRIDE should be connected to the type and the instance entity. have we moved to a situation that assumes the type applicability based on the instance value seems a bit woolly to be to not use explicit notation in the standard??

I got a response about this issue. It was said that `PSET_TYPEDRIVENOVERRIDE` would be documented only on occurrences and consequently the applicabilities followed suit.

**Comment by AlexBrad1eyCT** [comment]
So Woolly it is...

**Comment by AlexBrad1eyCT** [comment]
Type Applicability Applied

File: IFC4x3/Properties/e/EnergySource_0yj3k0qSOHuO00025QrE$V/Documentation.md [hunk 4]
```diff
@@ -1 +1 @@
-Enumeration defining the energy source or fuel cumbusted to generate heat.
+Enumeration defining the energy source or fuel combusted to generate heat.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/EnergySource_0yj3k0qSOHuO00025QrE$V/Documentation.md:1]: 😄

File: IFC4x3/PropertyEnumerations/b/PEnum_BoreholeState.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="PEnum_BoreholeState" UniqueId="84d3d099-2c7c-40b5-97f4-514b9ecd836a">
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_BoreholeState" Name="PEnum_BoreholeState" UniqueId="84d3d099-2c7c-40b5-97f4-514b9ecd836a">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/b/PEnum_BoreholeState.xml:2]: This one requires an encoded guid in id. Re-serializing with IfcDoc would probably make sense.

Comment (SergejMuhic) [reply IFC4x3/PropertyEnumerations/b/PEnum_BoreholeState.xml:2]: I will merge and regenerate if necessary.

**Pull request merged**